docker/sapphire-localnet/Dockerfile

```
@@ -100,6 +112,7 @@ ENV OASIS_NODE_BINARY=/oasis-node
 ENV OASIS_NET_RUNNER_BINARY=/oasis-net-runner
 ENV OASIS_NODE_DATADIR=/serverdir/node
 ENV OASIS_WEB3_GATEWAY_BINARY=/oasis-web3-gateway
+ENV OASIS_ENVOY_BINARY=envoy
```

I would prefer absolute paths just to be sure.

```suggestion
ENV OASIS_ENVOY_BINARY=/usr/bin/envoy
```